docs/tutorials/running-local.md

```diff
@@ -5,5 +5,251 @@ description: Building a DApp using Nevermined frameworks
 
 # How to build a React application integrated with Nevermined
 
-To be completed
+The idea of this tutorial is give a quick go through about using the [Nevermined Catalog](https://github.com/nevermined-io/components-catalog) of React components to support the development of dApps using the Nevermined environment.
 
+## Prerequisites
+The tutorial assumes your familiarity with blockchain, and general programming.
+
+## Let's go!!
+
+Instructions for following along locally using your preferred text editor.
+
+### Setup the React app
+1. Make sure you have a recent version of [Node.js](https://nodejs.org/en/) installed.
+2. Follow the [installation instructions for Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) to make a new project. Or simply run `npx create-react-app my-nevermined-app --template typescript`.
+3. Move to the my-nevermined-app directory `cd my-nevermined-app`.
+4. Run `yarn add @nevermined-io/components-catalog` or `npm install --save @nevermined-io/components-catalog` depending of your favourite package manager.
+5. Run `yarn run start` and open http://localhost:3000 and you will see the progress.
+
+### Create your Nevermined dApp
+
+Once the react app is running and you have your catalog dependency, the next step will be to setup the configuration:
+
+1. Donwload [Nevermined abi](http://artifacts-nevermined-rocks.s3.amazonaws.com/80001/public/contracts_v2.0.0.tar.gz) and store them in a folder named contracts in the public folder.
```

Comment:
People don't know what are the ABIs, speak a bit about that and why they need locally